lib/bindConnectors/formatError.js

```diff
@@ -18,11 +22,24 @@ module.exports = function (response, event) {
 
 	if (_.isError(details)) {
 
-		details = {
+		var newDetailsObject = {
 			message: details.message || 'API error',
```

Comment:
I know it was here already, but maybe missing message should be regarded as connector error? And if so, the whole object can be provided to the backend

Reply:
Oh, what, change the default from `#api_error` to `#connector_error`?

Reply:
Not sure what you mean provide whole object to BE?

Reply:
`Oh, what, change the default from #api_error to #connector_error` not in general case, no. I mean only if message is missing. The reason is that message should be regarded as required (because without a message, it could be hard for the user or us to identify the error).
So:
```
if (!details.message) {
  newDetailsObject = {
     ...details,
     message: '--message is missing from the error--',
     code: '#connector_error',
  }
}
```

Reply:
Hmmm, and should `#connector_error` override if a `code` is defined?

Reply:
yeah probably, if message is missing. You could store the original code in the payload i guess, though it's probably so rare that may be not worth it